README.md

```diff
@@ -1,176 +1,79 @@
-2. You can embed the video in a web view. See our [embed documentation](https://developer.vimeo.com/apis/oembed) for more.
-3. PRO members can generate direct video file links for their own players. See our [API documentation](https://developer.vimeo.com/api) for more.
+2. Drop the VimeoDeeplink.swift (or VIMDeeplink.h/.m) file(s) into your project.
 
-If you wish to use this library, you must make sure your user has the Vimeo app installed. We expose a [method](#check-if-the-vimeo-app-is-installed) to verify this. If the app is not installed, you can send your users to the [app store](#open-app-in-the-app-store).
+# Background
+In order for deeplinks to work, the user must have the Vimeo iOS app installed on their device. We expose a [method](#check-if-the-vimeo-app-is-installed) to verify this. If the app is not installed, you can send your users to the [app store](#open-app-in-the-app-store). If the app is installed, you can deeplink til your :heart:'s content. 
 
-If the user has the app installed, you can open the video. First you must find your video's URI. This is the unique identifier provided by the Vimeo API. You can learn more at the [developer site](https://developer.vimeo.com/api). We will release an iOS API library in the future, but for the moment I recommend starting with an OAuth 2 library and working from there.
+Deeplinking to a specific piece of Vimeo content (e.g. a channel, category, user, or video object, etc.) requires that you have the object's `uri`. The `uri` is the unique identifier provided by the Vimeo API. You can obtain object `uri`s by making requests to the Vimeo API or by constructing them by hand (although this is not advised). You can learn more at the [developer site](https://developer.vimeo.com/api). The [VimeoNetworking](https://github.com/vimeo/VimeoNetworking) library for iOS facilitates authentication and communication with the Vimeo API.
 
-Once you have your video's URI, you can pass it into the [proper method](#view-a-video-in-the-vimeo-app) and the Vimeo app will open to your video.
+Once you have an object's `uri`, you can pass it to `VimeoDeeplink` and start linking.
 
-# View a user
+# Swift and Objc
 
-First you must make sure your user has the Vimeo app installed. We expose a [method](#check-if-the-vimeo-app-is-installed) to verify this. If the app is not installed, you can send your users to the [app store](#open-app-in-the-app-store).
+VimeoDeeplink provides both Swift and Objc APIs. They are almost identical in terms of functionality. There are a few Swift examples below. The full Objc API can be viewed in [VIMDeeplink.h](VimeoDeeplink/VimeoDeeplink/VIMDeeplink.h)/[.m](VimeoDeeplink/VimeoDeeplink/VIMDeeplink.m).
 
```

Comment:
Sorry, I meant the swift and obj-c files, not both obj-c m/h files. I know you linked to swift below, but it may be nice to link to it here too.